.rosinstall

```diff
@@ -0,0 +1,62 @@
+- git:
+    uri: 'https://github.com/ipa320/cob_android.git'
+    local-name: cob_android
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_calibration_data.git'
+    local-name: cob_calibration_data
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_command_tools.git'
+    local-name: cob_command_tools
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_common.git'
+    local-name: cob_common
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_control.git'
+    local-name: cob_control
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_driver.git'
+    local-name: cob_driver
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_environments.git'
+    local-name: cob_environments
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_extern.git'
+    local-name: cob_extern
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_hand.git'
+    local-name: cob_hand
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_robots.git'
+    local-name: cob_robots
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ros-industrial/ros_canopen.git'
+    local-name: ros_canopen
+    version: indigo-devel
+- git:
+    uri: 'https://github.com/ipa320/schunk_modular_robotics.git'
+    local-name: schunk_modular_robotics
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/SICKAG/sick_visionary_t.git'
+    local-name: sick_visionary_t
+    version: indigo-devel
+- git:
+    uri: 'https://github.com/ipa320/sick_flexisoft.git'
+    local-name: sick_flexisoft
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_docking.git'
+    local-name: cob_docking
+    version: indigo_dev
+
```

Comment:
the initial idea was just clone cob_robots and cob_calibration_data , the repositories that had a specific robot configuration. But  we are constantly developing software, we are not so fast releasing new features and also we need some private packages... but I dont think that all the cob_bringup dependencies are needed, let me recheck the list